beacon-chain/rpc/eth/config/handlers.go

```diff
@@ -124,11 +124,27 @@ func convertValueForJSON(v reflect.Value, tag string) interface{} {
 			if !v.Field(i).CanInterface() {
 				continue // unexported
 			}
-			key := f.Tag.Get("json")
-			if key == "" || key == "-" {
+			jsonTag := f.Tag.Get("json")
+			if jsonTag == "-" {
+				continue // skip fields with json:"-"
```

Comment:
```suggestion
				continue
```
This is already obvious from the code